### PR TITLE
insights

### PR DESCRIPTION
**Pull request by gandesc**

### CODE REVIEW

File: src/app/Exceptions/Role.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace LaravelEnso\Roles\app\Exceptions;
+
+use LaravelEnso\Helpers\app\Exceptions\EnsoException;
+
+class Role extends EnsoException
+{
+    public static function noConfigForAdmin()
```
Comment (aocneanu) [inline src/app/Exceptions/Role.php]: let's rename this to `adminConfig`

File: src/app/Exceptions/Role.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        return new self(__("You don't need a config file for admin"));
+    }
+
+    public static function noSyncForAdmin()
```
Comment (aocneanu) [inline src/app/Exceptions/Role.php]: and this to `adminSync`

File: src/routes/app/roles.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+Route::namespace('Role')
```
Comment (aocneanu) [inline src/routes/app/roles.php]: rollback.

Comment (gandesc) [reply src/routes/app/roles.php]: rolled back

**Pull request merged**